src/notes/notes.service.spec.ts

```diff
@@ -67,9 +76,800 @@ describe('NotesService', () => {
       .compile();
 
     service = module.get<NotesService>(NotesService);
+    noteRepo = module.get<Repository<Note>>(getRepositoryToken(Note));
+    revisionRepo = module.get<Repository<Revision>>(
+      getRepositoryToken(Revision),
+    );
+    userRepo = module.get<Repository<User>>(getRepositoryToken(User));
+    groupRepo = module.get<Repository<Group>>(getRepositoryToken(Group));
   });
 
   it('should be defined', () => {
     expect(service).toBeDefined();
   });
+
+  describe('getUserNotes', () => {
+    describe('works', () => {
+      const user = User.create('hardcoded', 'Testy') as User;
+      const alias = 'alias';
+      const note = Note.create(user, alias);
+
+      it('with one note', async () => {
```

Comment:
Title does not match content (no note is used)

Reply:
fixed